app/src/ui/multi-commit-operation/confirm-abort-dialog.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
+import { Dialog, DialogContent, DialogFooter } from '../dialog'
+import { OkCancelButtonGroup } from '../dialog/ok-cancel-button-group'
+
+interface IConfirmAbortDialogProps {
+  readonly operation: string
```

Comment:
From what I see above, it's expected this string to be capitalized, right? Probably worth to either (a) suggest that in a comment or (b) capitalize it correctly when it's used for macOS. What do you think?

Reply:
The way I am using it is by passing in this enum value. 
![image](https://user-images.githubusercontent.com/75402236/119159942-c49b4b00-ba25-11eb-912c-3cc004da8171.png)

And since it's noted to be in sentence case.. I figured that would be enough.

Reply:
>(b) because I will just be passing an enum and this is not likely something to be used for random input.. I don't think it is worth it to programmatically fix it.

Reply:
Added note in case we use it elsewhere.. maybe merge.. but I am halfway thinking merge could be refactored to use the multi-commit flow...